### PR TITLE
Hotfix latex

### PR DESCRIPTION
**Pull request by bcho892**
**Issue**

Small bug where latex didnt work on exam editor

### CODE REVIEW

File: client/src/components/app/DisplayText.vue [hunk 1]
```diff
@@ -82,13 +82,16 @@ export default {
       })
 
       this.htmlContent = this.converter.makeHtml(html)
-
+      this.renderLatex()
+    },
+    renderLatex() {
       this.$nextTick(() => {
         document.querySelectorAll('.latex').forEach((node) => {
+          console.log('fuck all here')
```
Comment (johnchen383) [inline client/src/components/app/DisplayText.vue:90]: remove plz ahah

**Review by johnchen383 — CHANGES_REQUESTED** [review]

**Comment by bcho892** [comment]
LOL

**Comment by bcho892** [comment]
was a feature trust

**Review by johnchen383 — APPROVED** [review]

**Review by Oculux314 — APPROVED** [review]
lgtm!

**Pull request merged**